### ISSUE TITLE
Page class "index" is meant to only target the homepage, but it's also targeting all sub-pages.

### ISSUE THREAD

**Issue by richard-child** [richard-child/richardwrites#2]
https://github.com/richard-child/richardwrites/blob/4dbe8f371e878737ac4b0ab96dff5f081c02532c/source/partials/_header.haml#L10

**Comment by michaelkoper** [comment]
Fixed bug where page_classes.include?(‘index’) was true when current page wasn’t the home page.

The problem was that page_classes is a string and not an array.

So this was true: `'a_fancy_page_index'.include?(‘index’)`

I added a new helper to make it a bit easier, so it not needed to do this in header: `if true ? 'selected' : ''`

In header_nav_selected_class I basically just convert the string to array `'a b c'.split(' ')` becomes `['a', 'b', 'c']`.

And in Ruby an array has the include? method which checks if the value exists in the array.

```
[‘fancy_page_index’].include?(‘index’) # => false
[‘fancy_page_index’].include?(‘fancy_page_index’) # => true
```
